package.json

```diff
@@ -27,7 +27,8 @@
     "zone.js": "^0.8.4",
     "bootstrap": "^3.3.7",
     "ngx-bootstrap": "^1.6.6",
-    "file-saver": "1.3.2"
+    "file-saver": "1.3.2",
+    "angular-2-dropdown-multiselect": "^1.3.2"
```

Comment:
Is the Angular 2 Dropdown Multiselect finally used? If not, better remove it from package.json